src/fft.js

```diff
@@ -472,6 +472,17 @@ define(function (require) {
     }
     this.analyser.smoothingTimeConstant = s;
   };
+  
+  /**
+   * Changes the length of the resulting array, the bin size and in turn the fftSize of the analyser.
+   * 
+   * @method setBins
+   * @param {Number} bins    Must be a power of two between 16 and 1024.
+   */
+  p5.FFT.prototype.setBins = function(b) {
+    this.bins = b;
+    this.analyser.fftSize = b * 2;
```

Comment:
Similar to [this comment](https://github.com/processing/p5.js-sound/pull/176#discussion_r121283301), `this.bins` still has the potential to mismatch the `analyser.fftSize`, and that could be fixed with a getter.

and, I wonder if we need to add a new method for setBins, or if we could instead use a setter when we define the property?